chore(apm/php): update steps to verify apt repo signature (#18099)
newrelic/docs-website: src/content/docs/apm/agents/php-agent/installation/php-agent-installation-ubuntu-debian.mdx

  Run the commands in this procedure as root.
</Callout>

<Steps>
  <Step>
    ### Configure the New Relic apt repository [#configure-repo]

>     Complete these two tasks:

Author: ```suggestion
    New Relic's apt repository configuration is only required once per system. To do so, complete these two tasks:
```

>     1. Add `http://apt.newrelic.com/debian/` apt repository to `sources.list` (only required once per system):

Author: ```suggestion
    1. Add `http://apt.newrelic.com/debian/` apt repository to `sources.list`:
```

        ```bash
        echo 'deb [signed-by=/usr/share/keyrings/download.newrelic.com-newrelic.gpg] http://apt.newrelic.com/debian/ newrelic non-free' | sudo tee /etc/apt/sources.list.d/newrelic.list
        ```

        This command adds `deb [signed-by=/usr/share/keyrings/download.newrelic.com-newrelic.gpg] http://apt.newrelic.com/debian/ newrelic non-free` to `/etc/apt/sources.list.d/newrelic.list`. It creates the file if it does not exist.

>     2. Execute the following to register New Relic as an authenticated source where apt-get will look for new packages. To get the New Relic public apt-key from global key servers, you need execute this command as root:

Author: ```suggestion
    2. Trust the New Relic GPG key:
```

        ```bash
        wget -O- https://download.newrelic.com/548C16BF.gpg | sudo gpg --dearmor -o /usr/share/keyrings/download.newrelic.com-newrelic.gpg
        ```

>         <Callout variant="tip">
>           If you don't run this command as root, you may see an error message about the public key.
>         </Callout>

Author: ```suggestion
       This command installs New Relic's public key used by `apt` to verify signatures of packages in `http://apt.newrelic.com/debian/` apt repository added in previous step.
```

        ```bash
        sudo newrelic-install install
        ```
  </Step>
  <Step>
    ### Configure your application name and New Relic license key [#license-key]

>         The installation process prompts for your application name and license key. Your <InlinePopover type="licenseKey" /> appears in the account information section of <DNT>**Account settings**</DNT>. For headless installations, you can also [insert your license key and application name by using `debconf`](#preseeding).

Author: ```suggestion
        The installation process prompts for your application name and license key. Your <InlinePopover type="licenseKey" /> appears in the account information section of <DNT>**Account settings**</DNT>. For headless installations, you can also [preconfigure your license key and application name by using `debconf`](#preseeding).
```

        ```bash
        dpkg -i newrelic-php5-common_X.X.X.X_all.deb newrelic-daemon_X.X.X.X_amd64.deb newrelic-php5_X.X.X.X_amd64.deb
        ```
  </Step>
  <Step>
    ### Configure your application name and New Relic license key [#configure-key]

>         The installation process prompts for your application name and license key. Your <InlinePopover type="licenseKey" /> appears in the account information section of <DNT>**Account settings**</DNT>. For headless installations, you can also [insert your license key and application name by using `debconf`](#preseeding).

Author: ```suggestion
        The installation process prompts for your application name and license key. Your <InlinePopover type="licenseKey" /> appears in the account information section of <DNT>**Account settings**</DNT>. For headless installations, you can also [pre-configure your license key and application name by using `debconf`](#preseeding).
```

  </Step>
  <Step>
    ### Restart your web server or FastCGI Process Manager (FPM) [#restart-server]

        Generate traffic for your app, and wait a few minutes for your application to send data to New Relic. Then, [check your app's performance in the New Relic UI](/docs/apm/applications-menu/monitoring/apm-overview-page).
  </Step>
</Steps>

<InstallFeedback />

> ## Headless: Insert license key and app name [#preseeding]

Author: ```suggestion
## Headless: Preconfigure license key and app name [#preseeding]
```

> For headless installations, you need to take steps to insert your <InlinePopover type="licenseKey" /> and [application name](/docs/site/naming-your-application). You must use the default php5 packages provided by your distribution. To insert the license key and application name, set values in your `debconf` database by using [`debconf-set-selections`](http://manpages.ubuntu.com/manpages/trusty/en/man1/debconf-set-selections.1.html).

Author: ```suggestion
For headless installations, you need to take steps to preconfigure your <InlinePopover type="licenseKey" /> and [application name](/docs/site/naming-your-application). You must use the default php5 packages provided by your distribution. To preconfigure the license key and application name, set values in your `debconf` database by using [`debconf-set-selections`](https://manpages.debian.org/bookworm/debconf/debconf-set-selections.1.en.html).
```

  </tbody>
</table>

> For example, you can run these commands to insert your app name and license key:

Author: ```suggestion
For example, you can run these commands to preconfigure your app name and license key:
```
